KubeArmor/feeder/feeder.go

```diff
@@ -711,3 +738,18 @@ func (fd *Feeder) PushLog(log tp.Log) {
 		}
 	}
 }
+
+func loadTLSCredentials(ip string) (credentials.TransportCredentials, error) {
+	// create certificate configurations
+	serverCertConfig := cert.DefaultKubeArmorServerConfig
+	serverCertConfig.IPs = []string{ip}
+	serverCertConfig.NotAfter = time.Now().Add(365 * 24 * time.Hour) //valid for 1 year
+	// as of now daemonset creates certificates dynamically
+	tlsConfig := cert.TlsConfig{
+		CertCfg:      serverCertConfig,
+		CertProvider: cfg.GlobalCfg.TLSCertProvider,
+		CACertPath:   CACertPath,
```

Comment:
```suggestion
		CACertPath:   cert.CertPath{
	                                     Base:     cfg.GlobalCfg.TLSCertPath,
	                                     CertFile: "ca.crt",
	                                     KeyFile:  "ca.key",
                                           },
```

Are these variables being used somewhere else? Why can't we do this?

Reply:
they are not being used anywhere else, we can do this.

Reply:
Or just create a function in cert module saying
```go
func GetCACertPath(){
return cert.CertPath{
	                                     Base:     cfg.GlobalCfg.TLSCertPath,
	                                     CertFile: "ca.crt",
	                                     KeyFile:  "ca.key",
                                           }
}
```

This will always be using latest value of `cfg.GlobalCfg.TLSCertPath` whenever called

Reply:
or 
```golang
func GetCACertPath(basePath string){
return cert.CertPath{
	                                     Base:    basePath,
	                                     CertFile: "ca.crt",
	                                     KeyFile:  "ca.key",
                                           }
}
```

then we can use it across all applications i.e. relay, karmor

Reply:
Yeah makes sense